browser/js/w2ui-1.5.rc1.js

```diff
@@ -6423,6 +6428,222 @@ w2utils.event = {
                 // event after
                 this.trigger($.extend(edata, { phase: 'after' }));
             }
+        },
+        getDocType: function(map) {
```

Comment:
I have to think how to avoid code duplication here...

Reply:
I have tried but at this stage we are not getting any doc type information from map object so will need to use this function for getImage url

Reply:
I see, can we then do it this way:
- we are not ready, map doctype is not set -> do what we did in the past, no change
- we are ready and we have doctype in map -> use that and setup icons using new aproach

Reply:
i have already done that and we had discussion like if we need to override the path using js that then browser will  take the old priority image path from css. 

once the image path is set to in DOM at run time ithen to change that conditionally  using JS not working well and icons still remains same path as define in CSS file.

Reply:
yes, I understand, but I mean in this function. when we detect that docType is available -> we add `style=""`, if docType is not available using map object -> we doon't add `style=""`

Reply:
Okay i will try this way

Reply:
@eszkadev now docType code duplication is removed. we are using doctype from map property it self. Thanks for that suggestion :)